spec/controllers/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+describe BookSuggestionsController do
+  describe 'POST #create' do
+    context('when user is authenticated') do
+      include_context 'Authenticated User'
+
+      context 'and creating a valid book suggestion' do
+        subject(:book_s) { build(:book_suggestion, user: user) }
+        let(:http_request) { post :create, params: { book_suggestion: book_s.attributes } }
+
+        it 'creates a new book suggestion' do
+          expect { http_request }.to change { BookSuggestion.count }.by(1)
+        end
+
+        it 'responds with OK status' do
+          http_request
+          expect(response).to have_http_status(:created)
+        end
+
+        it 'sets user id value' do
+          http_request
+          expect(response_body['user_id']) =~ user.id
```

Comment:
This is not the correct way to test the user value. You should compare it: `expect(actual).to eql(expected)` in this case: `expect(response_body['user_id'])to eql(user.id)`